recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ package:
 
```

Comment:
We did something special here with the `patch_level` which impacts the build scripts' interaction with maven. Has anyone checked whether this is still correct? This may be accidentally working because the `patch_level` set here in line 5 happens to coincide with the one in the version number.

Reply:
it happens fairly frequently with the way the upstream releases occur... i'll take a look...